src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_TestPage.xaml.cs

```diff
@@ -0,0 +1,381 @@
+
+				this.DropDetailsTextBlock.Visibility = Visibility.Collapsed;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Html))
+			{
+				title += " (Html)";
+
+				var html = await args.DataView.GetHtmlFormatAsync();
+				details = html;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Rtf))
+			{
+				title += " (Rtf)";
+
+				var rtf = await args.DataView.GetRtfAsync();
+				details = rtf;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.StorageItems))
+			{
+				title += " (StorageItems)";
+
+				var items = await args.DataView.GetStorageItemsAsync();
+				if (items.Count > 0)
+				{
+					// Only use the first item
+					foreach (IStorageItem item in items)
+					{
+						var file = item as StorageFile;
+
+						if (file != null)
+						{
+							details += "Name: " + file.Name + Environment.NewLine;
+							details += "DisplayName: " + file.DisplayName + Environment.NewLine;
+							details += "DisplayType: " + file.DisplayType + Environment.NewLine;
+							details += "ContentType: " + file.ContentType + Environment.NewLine;
+							details += "Path: " + file.Path;
+						}
+					}
+				}
+			}
+			else if (args.DataView.Contains(StandardDataFormats.Text))
+			{
+				title += " (Text)";
+
+				var text = await args.DataView.GetTextAsync();
+				details = text;
+			}
+			else if (args.DataView.Contains(StandardDataFormats.UserActivityJsonArray))
+			{
+				title += " (UserActivityJsonArray)";
+
+				// TODO
+			}
+			else if (args.DataView.Contains(StandardDataFormats.WebLink))
+			{
+				title += " (WebLink)";
+
+				var webLink = await args.DataView.GetWebLinkAsync();
+				details = webLink.ToString();
+			}
+
+			this.DropTitleTextBlock.Text = title ?? string.Empty;
+			this.DropDetailsTextBlock.Text = details ?? string.Empty;
+
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when a drag operation leaves the <see cref="DropBorder"/>.
+		/// </summary>
+		private void DropBorder_DragEnter(object sender, DragEventArgs e)
+		{
+			// Accept everything possible
+			if (e.DataView.RequestedOperation != DataPackageOperation.None)
+			{
+				e.AcceptedOperation = e.DataView.RequestedOperation;
+			}
+			else
+			{
+				e.AcceptedOperation = DataPackageOperation.Copy | DataPackageOperation.Link | DataPackageOperation.Move;
+			}
+
+			this.ShowDragDropDetails(nameof(UIElement.DragEnter), e);
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when a drag operation leaves the <see cref="DropBorder"/>.
+		/// </summary>
+		private void DropBorder_DragLeave(object sender, DragEventArgs e)
+		{
+			this.ShowDragDropDetails(nameof(UIElement.DragLeave), e);
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when a drag operation is over the <see cref="DropBorder"/>.
+		/// </summary>
+		private void DropBorder_DragOver(object sender, DragEventArgs e)
+		{
+			this.ShowDragDropDetails(nameof(UIElement.DragOver), e);
+			return;
```

Comment:
Codacy found an issue: [Remove this redundant jump.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270104)